<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableViewWidget</class>
 <widget class="QWidget" name="TableViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CustomTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>671</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ContiguousSelection</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="interpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filterButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>ui.widgets.CustomTableWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
